app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -12,25 +14,25 @@
 public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
 
     private Button loginButton;
+    private Button signUpButton;
     private EditText email;
     private EditText password;
 
     @Override
     public void init() {
         loginButton = (Button) getView().findViewById(R.id.vLoginButton);
+        signUpButton = (Button) getView().findViewById(R.id.vSignUpButton);
         email = (EditText) getView().findViewById(R.id.vLoginEmail);
         password = (EditText) getView().findViewById(R.id.vLoginPassword);
+        TextView termsAndConditionsLink = (TextView) getView().findViewById(R.id.vLoginTermsAndConditionsLink);
```

Comment:
you should "declarate" this `termsAndConditionsLink` as private attribute of the fragment and then initialise it here